src/main/java/frc/trigon/robot/subsystems/intake/IntakeCommands.java

```diff
@@ -0,0 +1,49 @@
+package frc.trigon.robot.subsystems.intake;
+
+import edu.wpi.first.wpilibj2.command.*;
+import frc.trigon.robot.RobotContainer;
+
+public class IntakeCommands {
+    public static Command getSetTargetStateAndWaitCommand(IntakeConstants.IntakeState intakeState) {
+        return new SequentialCommandGroup(
+                getSetTargetStateCommand(intakeState),
+                getWaitCommand().andThen(getStopIntakeCommand())
+        );
+    }
```

Comment:
this should only be called for the collection command... 

This should look like this:
First, call this functional command: 
* OnInit, set the target voltage. 
* OnEnd, send a StaticBrake request.
* IsFinished points to the hasNote. 

Then, call a wait command.
Then, call an instant command to stop the motor.